src/templates/weekEnded/html.mjml

```diff
@@ -1,41 +1,85 @@
+    <mj-preview>{{{'This is an automated email you requested to let you know when the week ends'}}}</mj-preview>
     <mj-attributes>
-      <mj-all font-family="Helvetica, Arial, sans-serif" color="#444444" />
+      <mj-all font-family="Roboto, Helvetica, Arial, sans-serif" color="#55575d" vertical-align="middle" text-align="center" background-color="#F4F4F4" />
     </mj-attributes>
     <mj-style inline="inline">
       .container {
-        max-width: 600px;
+      max-width: 600px;
       }
-
       .link {
-        text-decoration: none;
-        font-size: 16px;
-        color: #346df1;
+      font-family: Roboto Helvetica, Arial, sans-serif;
+      text-decoration: none;
+      font-size: 16px;
+      color: #346df1;
+      }
+      .footerLink {
+      font-size: 14px;
+      color: #000000;
+      text-decoration: none;
+      color: #346df1;
       }
     </mj-style>
   </mj-head>
   <mj-body>
-    <mj-section background-color="#F9F9F9" full-width="full-width">
-      <mj-column width="100%" css-class="container">
-        <mj-text align="center" padding="0 0 25px 0" font-size="22px">
-          <strong>{{ domain }}</strong>
-          &nbsp; | &nbsp;
-          <a href="{{ browserLink }}" class="link">View online version</a>
+
+  <mj-section padding="0px 0px 15px 0px">
+      <mj-column vertical-align="middle">
+        <mj-text align="left" font-size="15px" padding="0px">
+          <p style="text-align: center; margin: 10px 0px;">
+            <span>Email not displaying correctly? </span>
+            <a style="font-size: 15px;" href="{{ browserLink }}" class="link">View it in your browser</a>
+          </p>
+        </mj-text>
+      </mj-column>
+    </mj-section>
+
+    <mj-section background-repeat="no-repeat" background-size="100% 100%" background-url="{{ domain }}/bkgd-pitch.png" padding="25px 200px" border-radius="10px 10px 0px 0px" >
+      <mj-column background-color="transparent">
+        <mj-image align="center" alt="Football Image" border="none" height="auto" src="{{ domain }}/football.svg" title="Football Image" width="150px"></mj-image>
+      </mj-column>
+      </mj-section>
+
+       <mj-section padding="0px" text-align="center">
+      <mj-column background-color="#ffffff" width="100%">
+        <mj-text padding="15px 0px 0px 0px">
+          <h1 align="center">
+            <span style="letter-spacing:.5px;">
+              <strong>NFL Confidence Pool</strong>
+            </span>
+          </h1>
         </mj-text>
       </mj-column>
+
       <mj-column width="100%" background-color="#fff" border-radius="10px">
-        <mj-text padding="30px 0 15px 0" font-size="18px" align="center">
-          <h1>Hello {{ user.userFirstName }},</h1>
+        <mj-text padding="10px 0px 10px 0px" font-size="18px" align="center">
+          <p>Hello {{ userFirstName }},</p>
+          <p>Week {{ week }} has just ended with
+{{ visitorTeam }} @ {{ homeTeam }}.</p>
```

Comment:
@balexander82 I don't think this will work since the teams here are objects.  Notice how in the original, we had to use homeTeam.teamShortName, for instance